pkg/sql/codegen/couler/codegen.go

```diff
@@ -81,6 +82,8 @@ func GenCode(programIR []ir.SQLFlowStmt, session *pb.Session) (string, error) {
 		DataSource:  session.DbConnStr,
 		StepEnvs:    stepEnvs,
 		WorkflowTTL: workflowTTL,
+		OSSSK:       os.Getenv("SQLFLOW_OSS_SK"),
+		SecretName:  os.Getenv("SQLFLOW_WORKFLOW_SECRET_NAME"),
```

Comment:
`SQLFLOW_WORKFLOW_SECRET_NAME` seems only used for the OSS key, maybe the k8s resource name can be a constant.

Reply:
Good point, I use a json string to configure the secret object.